releasenotes/notes/process-discovery-check-c4662c390293827b.yaml

```diff
@@ -0,0 +1,9 @@
+upgrade:
+  - |
+    Process Agent now runs by default even outside of a container. When running, the process agent does not collect full
+    statistics on proceses. Instead, the process agent only gathers metadata on processes, and uses this data to suggest
+    possible integrations that the user may enable. This check is lightweight, and is configured by default to only run every 4 hours.
```

Comment:
Nit: this seems a bit redundant for release note, maybe just keep the `features` section?

Reply:
Sure, however I am worried that process agent running by default outside of a container could be considered a breaking change to some customers (especially if they don't want processes to be collected on that machine at all). How about if I change it to just `Process Agent now runs by default even outside of a container.`?

Let me know what you think.

Reply:
Ah, I thought we'd need additional changes for that, this isn't actually the case with this PR in particular, because it won't start by default, unless the new config change affects that?

Reply:
The new config change should make it start by default. Should we change that?

Reply:
If we are setting the default to `false` in this version we should drop the upgrade section.